src/components/shared/Entity/Shard/Information.tsx

```diff
@@ -0,0 +1,175 @@
+
+    const tableHeaderFooterSx: SxProps<Theme> = {
+        background: theme.palette.background.paper,
+    };
+
+    return (
+        <Stack direction="column" spacing={2} sx={{ m: 2 }}>
+            <Stack direction="row" spacing={1}>
+                <Typography
+                    component="span"
+                    variant="h6"
+                    sx={{
+                        alignItems: 'center',
+                    }}
+                >
+                    <FormattedMessage id="detailsPanel.shardDetails.title" />
+                </Typography>
+                <ExternalLink
+                    link={intl.formatMessage({
+                        id: 'detailsPanel.shardDetails.docPath',
+                    })}
+                >
+                    <FormattedMessage id="terms.documentation" />
+                </ExternalLink>
+            </Stack>
+
+            {error ? (
+                <AlertBox
+                    severity="error"
+                    short
+                    title={
+                        <FormattedMessage id="detailsPanel.shardDetails.fetchError" />
+                    }
+                >
+                    <MessageWithLink messageID="error.message" />
+                </AlertBox>
+            ) : (
+                <>
+                    <ShardErrors shards={taskShards} />
+
+                    <Grid item xs={12}>
+                        <TableContainer>
+                            <Table size="small">
+                                <TableHead>
+                                    <TableRow sx={{ ...tableHeaderFooterSx }}>
+                                        {columns.map((column, index) => (
+                                            <TableCell
+                                                key={`${column.field}-${index}`}
+                                            >
+                                                <Typography>
+                                                    {column.headerIntlKey ? (
+                                                        <FormattedMessage
+                                                            id={
+                                                                column.headerIntlKey
+                                                            }
+                                                        />
+                                                    ) : null}
+                                                </Typography>
+                                            </TableCell>
+                                        ))}
+                                    </TableRow>
+                                </TableHead>
+
+                                <InformationTableBody
+                                    shards={taskShards}
+                                    page={page}
+                                    rowsPerPage={rowsPerPage}
+                                    columns={columns}
+                                />
```

Comment:
This is the main change... we no longer check if there are shards before rendering at able so we can show a loading status.